src/cf-typography/src/atoms/links.less

```diff
@@ -147,7 +76,12 @@
                 left: 1em;
             }
 
-            &.a-link__before {
+            &.cf-icon__after {
+                padding-left: 1em;
+                padding-right: 2.25em;
+            }
+
+            &.cf-icon__before {
                 padding-left: 2.25em;
```

Comment:
With the new way icons are being handled, this line and the `padding-left: 1em;` in `&.cf-icon__after` can be deleted and the `padding-left: 1em` can be restored to the parent `&.a-link__bg` selector.